conda/meta.yaml

```diff
@@ -11,15 +11,16 @@ build:
   noarch: python
   preserve_egg_dir: True
   number: 2
-  script: {{PYTHON}} setup.py install #--single-version-externally-managed --record=record.txt
+  #script: {{PYTHON}} setup.py install #--single-version-externally-managed --record=record.txt
+  script: {{ PYTHON }} -m pip install .
 
 requirements:
   build:
     - python {{PY_VER}}
     - setuptools
     - openalea.deploy
   run:
-    - python >=3.6
+    - python >=3.10
```

Comment:
Is Python 3.10 mandatory? Or can we have Python 3.7 or 3.8 as a minimal version?